compiler/rustc_target/src/spec/mod.rs

```diff
@@ -1687,6 +1687,7 @@ supported_targets! {
 
     ("armebv7r-none-eabi", armebv7r_none_eabi),
     ("armebv7r-none-eabihf", armebv7r_none_eabihf),
+    ("armv7-rtems-eabihf", armv7_rtems_eabihf),
```

Comment:
I honestly cannot infer the what the overall grouping pattern is in this file, ...

... but I think you'd be safer if you put this off in a line on its own (with newlines separating it from other groups) rather than embedding in in the middle of this sequence of "arm*-none-eabi[hf]" entries.